### DISCUSSION TITLE
Non-human animal diseases

### DISCUSSION THREAD

**Discussion by ImkeTammen** [monarch-initiative/mondo#5630] [General]
I am a co-curator of Online Mendelian Inheritance in Animals (OMIA, https://www.omia.org/home/) and we are in the process to creating hyperlinks from OMIA to Mondo and want to add OMIA information to MONDO.
 
At the moment the ‘non-human animal disease’ Mondo section under ‘disease or disorder’ is clearly under development and  seems to be structured by ‘species’  (e.g. cat disease) or ‘species group’ (bird disease) [I would like to propose to link these to NCBI taxonomy species terms (which include a common name) instead of  creating new ‘species’ or ‘species group’ terms/definitions] or by disease (e.g. prion disease, non-human animal).
 
Can anyone comment how the ‘non-human animal disease’ Mondo term will be structured in the long term (e.g. species / infectious / inherited (single locus and/or genetic susceptibility)/ nutrition related disease / neoplasm or by biological system etc), and how in the future links between orthologous inherited diseases in humans and non-human animals will be made / will be displayed in MONDO?

Kind regards,

Imke

**Comment by sabrinatoro** [comment]
The current plan is : 
- 3 main branches in Mondo:
     -  human specific disease
     -  non-human animal disease
     - infectious disease (which will include diseases that are specific to human, specific to non-human animal, and infectious diseases that could affect either human and non-human animals)

- there will be several disease classifications within the non-human animal disease branch (this is possible because terms could have multiple parents)
    - classification based on species (referring to NCBITaxon for the species) 
    - classification based on the "type of disease" (e.g. neoplasm, nervous system disorder,...) which will be similar to the classification of diseases in human. 

- whenever possible, we will add a relation between the non-human animal disease and the analogous human disease. 

We will review this plan as we are adding new non-human diseases into Mondo (@katiermullen will bring her experience and knowledge as a vet)
